webrtc/guides/callFlow.md

```diff
@@ -0,0 +1,31 @@
+## Call Flow
+
+In this diagram we will review the interactions you will need in order to establish a working WebRTC session. To simplify the diagram, we are showing only one web browser in the session, but adding more merely follows the same pattern.
+
+
+![alt_text](../../images/webrtc_callflow1.png "image_tooltip")
+
+
+We start by creating a Session (1) and a Participant (2), and then adding the Participant to the Session (3) - when we do so, we will be specifying a Subscription to the SessionId within the _body_ parameter of the addParticipantToSession() call.
+
+Next we serve a page to the user that has the required WebRTC components as well as the JWT that we received when we [made what call???].
+
+Within our browser javascript implementation, we will be calling connect() (5) in order to establish our connection to the WebRTC platform. Once that is complete, we can call publish() (6) to start flowing media. 
+
+When media is flowing to us, an onStreamAvailable listener will be triggered. [what do we do then?]
+
+To remove this participant from the call, because the call has ended or just because just this participant is done, we call removeParticipantFromSession() (8) on the server side. If we only wanted to remove some subscriptions from this Participant, but leave them in the session otherwise, we can just call [WHAT?] from the server application.
+
+At the conclusion of the session, the browser javascript should call disconnect() (9) to sever the websocket connection with the WebRTC platform. Once all other participants have similarly been disconnected, the server application can call deleteSession (10) for proper hygiene. All Sessions are concluded and cleaned up, along with associated Subscriptions, after 9 hours.
+
```

Comment:
I think it would be a good idea to have a heading here describing the diagram and text that follows, calling it something like "Creating a Call" or "Call Initiation"